.gitignore

```diff
@@ -8,7 +8,6 @@ Makefile.in
 /missing
 /libtool
 /ltmain.sh
-/lib/*
```

Comment:
Now that you mention. I don't know how this made it in here, but it makes perfect sense to remove it.